src/components/Checkbox/Checkbox.js

```diff
@@ -1,43 +1,37 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
-import Icon from '../Icon';
 
 const Checkbox = ({
   className,
   id,
   labelText,
   onChange,
-  iconDescription,
+  indeterminate,
```

Comment:
Should we default this prop to `false`? Or should we just add a check to the ref method that `indeterminate` is not `undefined` before applying it?